node-registrar/pkg/server/handlers.go

```diff
@@ -783,3 +793,20 @@ func ensureOwner(c *gin.Context, twinID uint64) {
 		return
 	}
 }
+
+// Helper function to validate timestamp hint
+func validateTimestampHint(timestampHint int64) error {
+	// Get the current timestamp in seconds
+	now := time.Now().Unix()
+
+	// Calculate acceptable range
+	lowerBound := now - min(now, UptimeReportTimestampHintDrift)
```

Comment:
time ops to use time.Add, time.Sub, time.Since,.. etc not +/- .. etc

Reply:
I was working directly with Unix timestamps (integers) instead of `time.Time` objects.
I updated the code to use `time.Time` and its operations.